### PR TITLE
LPS-112480 Apply new styles to ckeditor

### PR DESCRIPTION
**Pull request by carloslancha**
We want to apply new styles defined by Lexicon to CKEditor. This is still wip, do not forward it.

https://www.figma.com/file/hwpqSFwApYdFrJfkS8oNWX/lexi-873-ckeditor-visual-style?node-id=68%3A4

### CODE REVIEW

File: modules/apps/frontend-css/frontend-css-web/src/main/resources/META-INF/resources/portal/_editor_ckeditor.scss [hunk 1]
```diff
@@ -1,3 +1,108 @@
...
+	border: 1px solid $secondary-l3;
+	border-radius: 4px;
+
+	.cke_inner {
+		border-radius: 4px;
+
+		.cke_top {
+			background-color: $white;
+			border-bottom: 1px solid $secondary-l3;
+			//border-radius: 4px;
```
Comment (marcoscv-work) [inline modules/apps/frontend-css/frontend-css-web/src/main/resources/META-INF/resources/portal/_editor_ckeditor.scss]: ninja border-radius?

Comment (carloslancha) [reply modules/apps/frontend-css/frontend-css-web/src/main/resources/META-INF/resources/portal/_editor_ckeditor.scss]: ninja!

Comment (carloslancha) [reply modules/apps/frontend-css/frontend-css-web/src/main/resources/META-INF/resources/portal/_editor_ckeditor.scss]: wip, remember xD

File: modules/apps/frontend-css/frontend-css-web/src/main/resources/META-INF/resources/portal/_editor_ckeditor.scss [hunk 2]
```diff
@@ -1,3 +1,108 @@
...
+
+						&.cke_button_on {
+							background-color: rgba($dark, 0.06);
+
+							&:last-child:after {
+								top: 0;
+							}
+						}
+
+						&:hover {
```
Comment (marcoscv-work) [inline modules/apps/frontend-css/frontend-css-web/src/main/resources/META-INF/resources/portal/_editor_ckeditor.scss:46]: probably we need to cover other states like focus:

```
&:hover,
&:focus
...
```

We should review all of them by focusing, press button, mark them as visited, etc

File: modules/apps/frontend-css/frontend-css-web/src/main/resources/META-INF/resources/portal/_editor_ckeditor.scss [hunk 3]
```diff
@@ -1,3 +1,108 @@
...
+					a.cke_button {
+						border: none;
+						border-radius: 4px;
+						margin: 0 2px;
+						padding: 7px 8px;
+
+						&:last-child:after, &.cke_button_disabled:hover:last-child:after {
+							border-right: 1px solid $secondary-l3;
+							height: 24px;
+							right: -7px;
```
Comment (marcoscv-work) [inline modules/apps/frontend-css/frontend-css-web/src/main/resources/META-INF/resources/portal/_editor_ckeditor.scss:31]: If there is no other way.... >_< !!!

File: modules/apps/frontend-css/frontend-css-web/src/main/resources/META-INF/resources/portal/_editor_ckeditor.scss (hunk 2 above)
Comment (carloslancha) [reply modules/apps/frontend-css/frontend-css-web/src/main/resources/META-INF/resources/portal/_editor_ckeditor.scss:46]: I think you can't focus on ckeditor buttons, if you find out how PLEASE tell me

**Comment by carloslancha** [comment]
ci:foward

**Comment by carloslancha** [comment]
ci:forward

**Comment by carloslancha** [comment]
ci:test:sf

**Comment by carloslancha** [comment]
ci:test:sf

**Comment by carloslancha** [comment]
ci:test:sf

**Comment by marcoscv-work** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by marcoscv-work** [comment]
Pull request submitted to https://github.com/carloslancha/liferay-portal/pull/266. [See changes here.](https://github.com/marcoscv-work/liferay-portal/compare/3c0c578346d9~1...3c0c578346d9) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**